manheim_cloudmapper/ses/report.py

```diff
@@ -0,0 +1,179 @@
+import os
```

Comment:
@jleopold28 When we've worked through some of the other comments in this review, let's circle back about this module either in-person or on a call. There are a few patterns used here that are rather un-Pythonic that I'd like to talk about, and I'm also a bit confused/unsure about the multiple file IO and the passing of instance variables.

Reply:
I've worked through almost all the other comments on the review. Let me know when you have some time and we can discuss this